assets/js/main.js

```diff
@@ -0,0 +1,209 @@
+  for (let i = 0; i < project.techLanguage.length; i += 1) {
+    ul.appendChild(createListItem(project, i));
+  }
+  return ul;
+}
+
+function createTitle(project) {
+  const h3 = document.createElement('h3');
+  h3.classList.add('project-title');
+  h3.textContent = project.title;
+  return h3;
+}
+
+function createContent(project) {
+  const p = document.createElement('p');
+  p.textContent = project.content;
+  return p;
+}
+
+function createImageIot(project, name) {
+  const imageName = name.toLowerCase();
+
+  if (imageName === 'iot') {
+    const imgIot = document.createElement('img');
+    imgIot.alt = project.images.iot.alt;
+    imgIot.src = project.images.iot.path;
+    return imgIot;
+  }
+
+  if (imageName === 'close') {
+    const imgClose = document.createElement('img');
+    imgClose.alt = project.images.btnClose.alt;
+    imgClose.src = project.images.btnClose.path;
+    imgClose.id = project.images.btnClose.id;
+    return imgClose;
+  }
+  return false;
+}
+
+function createPopup(project) {
+  const projectHeaderDiv = document.createElement('div');
+  projectHeaderDiv.classList.add('project-header');
+  projectHeaderDiv.append(createTitle(project), createListTech(project));
+
+  const projectContentDiv = document.createElement('div');
+  projectContentDiv.classList.add('project-contet');
+  projectContentDiv.append(createContent(project), createContent(project));
+
+  const projectSeeDiv = document.createElement('div');
+  projectSeeDiv.classList.add('project-see');
+  projectSeeDiv.append(createSeeButtons(project, 0), createSeeButtons(project, 1));
+
+  const projectCardDiv = document.createElement('div');
+  projectCardDiv.classList.add('project-card');
+  projectCardDiv.append(projectHeaderDiv, projectContentDiv, projectSeeDiv);
+
+  const projectImgDiv = document.createElement('div');
+  projectImgDiv.classList.add('project-img');
+  projectImgDiv.append(createImageIot(project, 'close'), createImageIot(project, 'iot'));
+
+  const projectDiv = document.createElement('div');
+  projectDiv.classList.add('project');
+  projectDiv.append(projectImgDiv, projectCardDiv);
+
+  const popupDiv = document.createElement('div');
+  popupDiv.classList.add('popup');
+  popupDiv.append(projectDiv);
+
+  hideAll();
+  document.querySelector('.modal').classList.remove('hide');
+  document.querySelector('.modal').append(popupDiv);
+
+  const closeProjectBtn = document.querySelector('#projectClose');
+
+  closeProjectBtn.addEventListener('click', () => {
+    document.querySelector('.modal').removeChild(document.querySelector('.popup'));
+    document.querySelector('.modal').classList.add('hide');
+    hideAll();
+  });
+}
+
+const seeProjectList = document.querySelectorAll('.btnPopup');
+window.addEventListener('resize', handleResizeMobile);
+
+seeProjectList.forEach((seeProject) => {
+  seeProject.addEventListener('click', () => {
+    if (windowSizemobile < 768) {
+      createPopup(project);
+    }
+  });
+});
```

Comment:
You did well with your popup modal and you have a beautiful design, weldone @Graciano1997 👍 

Please implement the following changes, as instructed in the project requirement you are to make your design to be similar to the design given. While you tried in achieving that, you have some corrections to make :

1. In the desktop view, of your popup modal , you are having an issue with height and alignment of the popup which doesn't give you a good UI for your portfolio. 

Your design :


https://github.com/Graciano1997/portofolio-graciano/assets/20537712/4d88c005-11bc-4d67-b1a2-2c10d4a01b93


2. For the desktop view, add a margin at the bottom , so your text isn't cut out as it appears in your design 

Your design :
![image](https://github.com/Graciano1997/portofolio-graciano/assets/20537712/57aecfe1-fae6-4423-914d-27b9762ea74b)

3. For your mobile view, kindly use the right font-color for all your text and fix the upper part of the mobile version to be similar to our figmma design 👍 

Figma design :
![image](https://github.com/Graciano1997/portofolio-graciano/assets/20537712/f521feb6-1633-4429-af06-ba1614f3c52a)

Your design :
![image](https://github.com/Graciano1997/portofolio-graciano/assets/20537712/c5976989-5934-468e-9089-28126362d30e)

Reply:
sorry but it is taking some additional seconds to be rendered on the screen